code/API_definitions/EdgeCloud_LcM.yaml

```diff
@@ -0,0 +1,712 @@
+                                          the port has to be exposed on a specific network other then default.
+                    protocol:
+                      type: string
+                      enum:
+                        - TCP
+                        - UDP
+                        - ANY
+                      description: Defines the IP transport communication protocol i.e., TCP, UDP or ANY
+                    port:
+                      type: integer
+                      format: int32
+                      minimum: 1
+                      maximum: 65535
+                      description: Port number exposed by the component. OP may generate a dynamic port towards the UCs 
+                                          corresponding to this internal port and forward the client traffic from dynamic port to containerPort.
+                    interfaceName:
+                      type: string
+                      description:  Interface Name. Required only if application has to be attatched to a network other then default.
+                    visibilityType:
+                      description: Defines whether the interface is exposed to outer world or not i.e., external, or internal.
+                                          If this is set to "external", then it is  exposed to external applications otherwise it is exposed 
+                                          internally to edge application components within edge cloud. When exposed to external world, 
+                                          an external dynamic port is assigned for UC traffic and mapped to the internal containerPort
+                      type: string
+                      enum:
+                        - VISIBILITY_EXTERNAL
+                        - VISIBILITY_INTERNAL
+                minItems: 1                
+              numOfInstances:
+                type: integer
+                format: int32
+                description: Number of component instances to be launched. 
+              restartPolicy:
+                type: string
+                enum:
+                  - RESTART_POLICY_ALWAYS
+                  - RESTART_POLICY_NEVER
+                description: How the platform shall handle component failure                        
+      required:
+        - name
+        - virtType
+        - fileName
+        - repository
+        - componentSpec
+    CpuArchType:
+          type: string
+          enum:
+            - x86_64
+            - arm_64
+          description: CPU Instruction Set Architecture (ISA) E.g., Intel, Arm etc. 
+    InstantiatedApp:
+      description: Information about the instantiated application
+      type: object
+      properties:
+        appInstanceId:
+          $ref: '#/components/schemas/AppInstanceId'
+        edgeCloudNodeName:
+          $ref: '#/components/schemas/EdgeCloudNodeName'
+        uri:
```

Comment:
We should add some description on what the API user can expect from the "Uri" and how it can be used

Reply:
Agree! Once the pull request is merged, further documentation can be added